package.json

```diff
@@ -5,38 +5,36 @@
   "dependencies": {
     "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.2",
-    "@types/enzyme": "^3.9.1",
     "@types/enzyme-adapter-react-16": "^1.0.5",
+    "@types/enzyme": "^3.9.1",
     "@types/google-map-react": "^0.23.11",
     "@types/jest": "^24.0.11",
     "@types/node": "^11.12.0",
-    "@types/react": "^16.8.10",
     "@types/react-dom": "^16.8.3",
     "@types/react-router-dom": "^4.3.1",
+    "@types/react": "^16.8.10",
     "firebase": "^5.8.4",
     "google-map-react": "^1.1.4",
-    "react": "^16.8.3",
     "react-dom": "^16.8.3",
-    "react-router": "^4.3.1",
     "react-router-config": "^4.4.0-beta.6",
     "react-router-dom": "^4.3.1",
+    "react-router": "^4.3.1",
     "react-scripts": "2.1.5",
+    "react": "^16.8.3",
     "typescript": "^3.3.4000"
   },
   "scripts": {
-    "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "lint": "prettier \"src/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
+    "ci-lint": "prettier --check --loglevel error '{data_pipeline,.github,src,.vscode}/**/*.{json,css,scss,md}' && tsc --noEmit && eslint 'src/**/*.{js,jsx,ts,tsx}' --quiet",
     "datapipe": "npm run strapped && npm run filter",
+    "eject": "react-scripts eject",
     "filter": "node data_pipeline/filter.js",
-    "strapped": "npm run strapped-charity && npm run strapped-crisisline",
+    "lint": "prettier --write --loglevel warn '{data_pipeline,.github,src,.vscode}/**/*.{json,css,scss,md}' && tsc --noEmit && eslint 'src/**/*.{js,jsx,ts,tsx}' --quiet --fix",
```

Comment:
Does this script still exist to automatically fix problems?

Reply:
yes! eslint (with prettier configs) will fix js/ts files, then prettier will do the rest